### PR TITLE
feat: update alert/report icons and column order

### PR DESCRIPTION
**Pull request by lilykuang**
### SUMMARY

- new icons for alert and report state: Success, Working, Error, Not triggered, On Grace
- Column reorder to put last execution time on first column
- update email and slack icon to lighter color and add right margin
- update slack icon svg

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

**Alert State:** 
Error
Screen Shot 2020-12-16 at 4 54 51 PM
Success
Screen Shot 2020-12-16 at 4 54 51 PM
On Grace
Screen Shot 2020-12-16 at 4 56 38 PM
Working
Screen Shot 2020-12-16 at 4 58 32 PM
Not Triggered
Screen Shot 2020-12-16 at 5 00 08 PM
**Report State**
Success
Screen Shot 2020-12-16 at 5 02 30 PM
Error
Screen Shot 2020-12-16 at 5 03 13 PM

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [x] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Comment by ktmud** [comment]
Is it just me or the size of the status icons looks kind of odd? The warning icon is too large while the others are too small.

**Comment by nytai** [comment]
It looks like it's only warning that's a bit larger. I'm guessing it's the underlying SVGs size and viewBox properties. This may be intentional, to draw attention to the failed ones? @mihir174 can you confirm?

**Comment by nytai** [comment]
Actually, they all look the same size in the designs 
Screen Shot 2020-12-16 at 1 48 59 PM

We should either add another variant of the icon `alert-solid-small` like we do with other icons or we can apply a `width` and `height` (and probably `position`) styles.

File: superset-frontend/src/views/CRUD/alert/components/AlertStatusIcon.tsx [hunk 1]
```diff
@@ -22,53 +22,87 @@ import { Tooltip } from 'src/common/components/Tooltip';
...
   return (
     <Tooltip title={lastStateConfig.label} placement="bottom">
       <StatusIcon
         name={lastStateConfig.name as IconName}
         status={lastStateConfig.status}
+        isReportEnabled={isReportEnabled}
+        viewBox={
+          lastStateConfig.name === 'alert-solid-small'
+            ? '-6 -6 24 24'
+            : '0 0 24 24'
```
Comment (lilykuang) [inline superset-frontend/src/views/CRUD/alert/components/AlertStatusIcon.tsx:104]: might be a better way to assign viewBox

**Review by nytai — APPROVED** [review]

File: superset-frontend/src/views/CRUD/alert/AlertList.tsx [hunk 2]
```diff
@@ -182,27 +184,33 @@ function AlertList({
...
-        }: any) =>
-          recipients.map((r: any) => (
-            <RecipientIcon key={r.id} type={r.type} />
-          )),
-        accessor: 'recipients',
-        Header: t('Notification Method'),
-        disableSortBy: true,
+        }: any) => {
+          return lastEvalDttm
+            ? moment.utc(lastEvalDttm).local().format(DATETIME_WITH_TIME_ZONE)
```
Comment (riahk) [inline superset-frontend/src/views/CRUD/alert/AlertList.tsx:204]: Unrelated but we should update other lists where we use `moment` to properly handle timezones

File: superset-frontend/src/views/CRUD/alert/components/AlertStatusIcon.tsx (hunk 1 above)
Comment (riahk) [reply superset-frontend/src/views/CRUD/alert/components/AlertStatusIcon.tsx:104]: Why do we need to do this, anyway? The viewbox on that icon is `0 0 24 24`, is there something strange going on with the spacing?

**Review by riahk — APPROVED** [review]

**Comment by lilykuang** [comment]
@riahk yea there is a strange spacing with `alert-solid-small` have to use `'-6 -6 24 24'` to fix it

**Pull request merged**